addons/ai/fnc_waypointGarrison.sqf

```diff
@@ -1,6 +1,6 @@
 #include "script_component.hpp"
 /* ----------------------------------------------------------------------------
-Script: fnc_waypointGarrison
+Function: CBA_fnc_waypointGarrison
```

Comment:
But that's wrong. There is no such function defined, as it not appears in CfgFunctions.
This script is used in CfgWaypoints, which, for BI knows why, uses paths to script files to operate.
"Script" instead of "Function" to indicate its usage as scheduled _script_ (as in, not a _function_, because no return value).

Reply:
Oh my, that's retarded. It is usable outside of `CfgWaypoints` though? Could be compiled under `CfgFunctions` in that case.

Reply:
I suppose it could be used as script outside of CfgWaypoints, but that would have to be in scheduled environment and would stall the current script instance, or... the "function" would have to be "spawned", which is the absolutely terrible BI way of doing things. Don't want that. Can't we just have the doc support "scripts" instead? Preferably ignore them, because proper code will only ever use them for such niche applications. Ignore as in just not list in the doc.

Reply:
Ignore sounds good.